lib/bloc/auth/auth_state.dart

```diff
@@ -0,0 +1,27 @@
+part of 'auth_bloc.dart';
+
+enum AuthStatus { unknown, authenticated, unauthenticated }
+
+class AuthState {
+  final AuthStatus authStatus;
+  final fbAuth.User? user;
+
+  AuthState({
+    required this.authStatus,
+    this.user,
+  });
+
+  factory AuthState.unknown() {
+    return AuthState(authStatus: AuthStatus.unknown);
+  }
+
+  AuthState copyWith({
+    AuthStatus? authStatus,
+    fbAuth.User? user,
+  }) {
+    return AuthState(
+      authStatus: authStatus ?? this.authStatus,
+      user: user ?? this.user,
+    );
+  }
+}
```

Comment:
In the tutorial he was also using equatable package. Everything works without adding that?

Reply:
It's works without :D but You have right